reporting/impl/src/main/resources/reports/templates/include/navheader.ftl

```diff
@@ -0,0 +1,9 @@
+            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-responsive-collapse">
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            <span class="wu-navbar-header">
+              <strong class="wu-navbar-header">Red Hat Application Migration Toolkit</strong>
+               - Web Console 
```

Comment:
Web Console? this is RHAMT CLI, right? Even we share that in Web UI, this is IMHO wrong to be here.